### PR TITLE
delete sctp shutdown status

### PR DESCRIPTION
**Pull request by LiangFang001**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/source/sctp/sctp_session.c [hunk 1]
```diff
@@ -283,19 +277,12 @@ STATUS sctp_session_free(PSctpSession* ppSctpSession)
     CHK(pSctpSession != NULL, retStatus);
 
     usrsctp_deregister_address(pSctpSession);
-    /* handle issue mentioned here: https://github.com/sctplab/usrsctp/issues/147
-     * the change in shutdownStatus will trigger onSctpOutboundPacket to return -1 */
-    ATOMIC_STORE(&pSctpSession->shutdownStatus, SCTP_SESSION_SHUTDOWN_INITIATED);
```
Comment (codingspirit) [inline src/source/sctp/sctp_session.c:288]: It's not safe to do so. sctp session need to wait a `ABORT ` packet before fully destroyed and `onSctpOutboundPacket ` might be called by libusrsctp

**Comment by ycyang1229** [comment]
Based on the information I got, you refer the PR, https://github.com/sctplab/usrsctp/issues/147 and https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/pull/1236. 
The usrsctp pr#147 mentioned usrsctp does not clear the void* pointer(or called address)of sctp session so the upper layer needs to protect this behavior in case the  conn_output uses the void* pointer. However, usrsctp may send the ABORT thing to complete the three way handshake. This pr may causes soft termination fails, and it also may cause accessing the wrong memory.

File: src/source/sctp/sctp_session.c (hunk 1 above)
Comment (LiangFang001) [reply src/source/sctp/sctp_session.c:288]: Hi,
Can `onSctpOutboundPacket` be triggered by `usrsctp_close`? I've tried many times, but the `onSctpOutboundPacket` never be called after `usrsctp_close`

**Comment by LiangFang001** [comment]
Hi,
@ycyang1229 @codingspirit 
Can 'onSctpOutboundPacket' be triggered by 'usrsctp_close'? I've tried many times, but the 'onSctpOutboundPacket' never be called after 'usrsctp_close'.

**Comment by LiangFang001** [comment]
Hi,
@ycyang1229 @codingspirit
Can `onSctpOutboundPacket` be triggered by `usrsctp_close`? I've tried many times, but the `onSctpOutboundPacket` never be called after `usrsctp_close`.

**Pull request closed without merging**